src/components/ITSBStore/itsbGraph.test.js

```diff
@@ -0,0 +1,69 @@
+import { describe, it, expect } from 'vitest';
+import { ITSBGraph } from './itsbGraph';
+
+import authors from '../../../data/authors.json';
+import places from '../../../data/places.json';
+import itineraries from '../../../data/itineraries.json';
+
+const initGraph = (authors, places, itineraries) => {
+  const graph = new ITSBGraph()
+  graph.init(authors, places, itineraries)
+  return graph;
+}
+
+describe('ITSBGraph', () => {
+
+  it('should load authors correctly', () => {
+    const graph = initGraph(authors.itemListElement, [], [])
+    const expected = graph.listAuthors();
+    expect(expected.length).toBe(12);
+  });
+
+  it('should load places correctly', () => {
+    const graph = initGraph([], places.features, []);
+    const expected = graph.listPlaces();
+    expect(expected.length).toBe(488);
+  });
+
+  it('should load itineraries correctly', () => {
```

Comment:
Could you add unit tests checking order preservation per each modification to the set of itineraries?

Reply:
Based on your other comments, I don't think this is a necessary unit test. Marking resolved